gmso/formats/top.py

```diff
@@ -136,9 +136,18 @@ def write_top(top, filename, top_vars=None):
                 _write_connection(top, angle, pot_types[angle.connection_type])
             )
 
+        angle_restraints = [angle for angle in top.angles if angle.restraint]
+        if angle_restraints:
+            out_file.write(
+                "\n[ angle_restraints ]\n"
+                ";\tai \taj \tai \tak \tfunct \ttheta_eq \tk \tmultiplicity \n"
+            )
+            for angle in angle_restraints:
+                out_file.write(_write_restraint(top, angle, "angle"))
+
         out_file.write(
             "\n[ dihedrals ]\n"
-            ";   ai     aj      ak      al  funct   c0      c1      c2\n"
+            ";\tai \taj \tai \tak \tfunct \tc0 \tc1 \tc2 \tc3 \tc4 \n"
```

Comment:
Unless I am mistaken, I think the 3rd entry here should be `ak` and not `ai`.

Reply:
And this should be 
```suggestion
            ";\tai \taj \tak \tal \tfunct \tc0 \tc1 \tc2 \tc3 \tc4 \n"
```

Reply:
Good catch